### ISSUE TITLE
E: unable to locate package libpam-aad  E: Unable to locate package libnss-aad

### ISSUE THREAD

**Issue by nhutwelker** [ubuntu/aad-auth#339]
### Is there an existing issue for this?

- [X] I have searched the existing issues and found none that matched mine

### Describe the issue

Cant install libpam-aad or libnss-aad

E: Unable to locate package libpam-aad
E: Unable to locate package libnss-aad

Tried the PPA 215 fix with  sudo  add-apt-repository ppa:justdenis/mantic-aad-proposed

`E: The repository 'https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'https://ppa.launchpadcontent.net/justdenis/mantic-aad-proposed/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
`

### Steps to reproduce it

steps to recreate 
`sudo add-apt-repository ppa:justdenis/mantic-aad-proposed`
`sudo apt update`

```shell

t:1 http://us.archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease   
Get:3 https://esm.ubuntu.com/apps/ubuntu jammy-apps-security InRelease [7,553 B]
Hit:4 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease 
Hit:5 http://us.archive.ubuntu.com/ubuntu jammy-security InRelease  
Ign:6 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security InRelease
Get:7 https://esm.ubuntu.com/apps/ubuntu jammy-apps-updates InRelease [7,459 B]
Hit:8 https://esm.ubuntu.com/infra/ubuntu jammy-infra-security InRelease                       
Hit:9 https://esm.ubuntu.com/infra/ubuntu jammy-infra-updates InRelease
Ign:10 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy InRelease
Ign:11 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security Release
Err:12 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy Release
  404  Not Found [IP: 185.125.190.52 443]
Ign:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata                                                                                      
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata                                                                                        
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons                                                                                         
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons                                                                                         
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons                                                                                       
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata                                                                                         
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata                                                                                       
Ign:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages                                                                                             
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages                                                                                               
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en                                                                                             
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US                                                                                          
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata                                                                                      
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata                                                                                        
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons                                                                                         
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons                                                                                         
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons                                                                                       
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata                                                                                         
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata                                                                                       
Ign:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages                                                                                             
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages                                                                                               
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en                                                                                             
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Err:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
  404  Not Found [IP: 185.125.190.52 443]
Ign:14 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Reading package lists... Done
E: The repository 'https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
root@ruby:/home/earthresit# sudo add-apt-repository ppa:justdenis/mantic-aad-proposed
Repository: 'deb https://ppa.launchpadcontent.net/justdenis/mantic-aad-proposed/ubuntu/ jammy main'
More info: https://launchpad.net/~justdenis/+archive/ubuntu/mantic-aad-proposed
Adding repository.
Press [ENTER] to continue or Ctrl-c to cancel.
Adding deb entry to /etc/apt/sources.list.d/justdenis-ubuntu-mantic-aad-proposed-jammy.list
Adding disabled deb-src entry to /etc/apt/sources.list.d/justdenis-ubuntu-mantic-aad-proposed-jammy.list
Adding key to /etc/apt/trusted.gpg.d/justdenis-ubuntu-mantic-aad-proposed.gpg with fingerprint E4A8F47E621D688B2898B0F0E2B4EA29315CBFCB
Hit:1 http://us.archive.ubuntu.com/ubuntu jammy InRelease
Hit:2 http://us.archive.ubuntu.com/ubuntu jammy-updates InRelease         
Hit:3 http://us.archive.ubuntu.com/ubuntu jammy-backports InRelease       
Hit:4 http://us.archive.ubuntu.com/ubuntu jammy-security InRelease                              
Ign:5 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security InRelease   
Hit:6 https://esm.ubuntu.com/apps/ubuntu jammy-apps-security InRelease            
Hit:7 https://esm.ubuntu.com/apps/ubuntu jammy-apps-updates InRelease             
Hit:8 https://esm.ubuntu.com/infra/ubuntu jammy-infra-security InRelease         
Hit:9 https://esm.ubuntu.com/infra/ubuntu jammy-infra-updates InRelease          
Ign:10 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy InRelease
Ign:11 https://ppa.launchpadcontent.net/justdenis/mantic-aad-proposed/ubuntu jammy InRelease
Ign:12 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security Release
Err:13 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy Release
  404  Not Found [IP: 185.125.190.52 443]
Err:14 https://ppa.launchpadcontent.net/justdenis/mantic-aad-proposed/ubuntu jammy Release
  404  Not Found [IP: 185.125.190.52 443]
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Ign:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Err:15 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 Packages
  404  Not Found [IP: 185.125.190.52 443]
Ign:16 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all Packages
Ign:17 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en
Ign:18 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse Translation-en_US
Ign:19 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all DEP-11 Metadata
Ign:20 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 DEP-11 Metadata
Ign:21 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 48x48 Icons
Ign:22 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64 Icons
Ign:23 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse DEP-11 64x64@2 Icons
Ign:24 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse amd64 c-n-f Metadata
Ign:25 https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy-security/multiverse all c-n-f Metadata
Reading package lists... Done
E: The repository 'https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: The repository 'https://ppa.launchpadcontent.net/justdenis/mantic-aad-proposed/ubuntu jammy Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

``` 

### Ubuntu users: System information and logs

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy

### Environment
* aad-auth version: please run `aad-cli version`
* Distribution: (**NAME** in `/etc/os-release`)
* Distribution version: (**VERSION_ID** on `/etc/os-release`):

### Log files
Please redact/remove sensitive information:
```raw
aad-auth logs can be found in the system journal and queried with:
`journalctl | grep _aad`
```

### Application settings
Please redact/remove sensitive information:
```raw
You can get the configuration file from /etc/aad.conf
```

### Double check your logs

- [X] I have redacted any sensitive information from the logs

**Comment by denisonbarbosa** [comment]
Hey, @nhutwelker! You are trying to install the packages in a Jammy (22.04) machine, but the package is available only on 23.04.

**Comment by sagipael** [comment]
i manage to get work the aad-auth module on U22
delete the created source.list  file.
and paste the below commands:

```
add-apt-repository  ppa:justdenis/aad-sudo-fix -y
rm -rf /etc/apt/sources.list.d/justdenis-ubuntu-aad-sudo-fix*
cat << EOT > /etc/apt/sources.list.d/justdenis-ubuntu-aad-sudo-fix-lunar.list
#This is a temporary PPA that contains a fixed version of aad-auth regarding sudo execution
#More info: https://launchpad.net/~justdenis/+archive/ubuntu/aad-sudo-fix

deb https://ppa.launchpadcontent.net/justdenis/aad-sudo-fix/ubuntu/ lunar main

EOT
```
